core/lib/Cardano/Address/Script.hs

```diff
@@ -229,8 +228,8 @@ scriptHashFromBytes bytes
 
 -- | Encode a 'ScriptHash' to bech32 'Text' or hex if key role is unknown.
 -- In the case of governance role, if one wants to include additional byte
--- as specified in [CIP-0129](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0129/README.md),
--- the function needs to be called with withByte=true.
+-- as specified in [CIP-0129](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0129/README.md)
+-- unless the function is called with (Just CIP0105).
```

Comment:
With the suggested change in `KeyHash` no need for `Just`

Reply:
done